### ISSUE TITLE
error: unable to recognize "STDIN": no matches for kind "StorageCluster" in version "ocs.openshift.io/v1alpha1"

### ISSUE THREAD

**Issue by jtaleric** [openshift-kni/install-scripts#103]
Params : 
```
[root@dhcp-19-130-242 OCS]# cat real_deploy.sh 
NAMESPACE="openshift-storage"
LOCALNAMESPACE="local-storage"

# export is required for envsubst
export rook_override="${rook_override:-true}"
export public_network="${public_network:-10.19.135.0/24}"
export cluster_network="${cluster_network:-172.22.0.0/24}"

# Provide disks to use for mon and osd pvcs
export cluster="${cluster:-mycluster}"

# Size number for mon pvcs
export mon_size="${mon_size:-5}"

# List of /dev/* disks to use for osd, separated by comma
export osd_devices="${osd_devices:-/dev/nvme0n1,/dev/nvme1n1,/dev/nvme2n1,/dev/nvme3n1}"

# Size number for osd pvcs
export osd_size="${osd_size:-55}"

```

Running the deploy-ocs.sh I get : 
```
[root@dhcp-19-130-242 OCS]# ./deploy-ocs.sh 
Using osd_devices /dev/nvme0n1,/dev/nvme1n1,/dev/nvme2n1,/dev/nvme3n1
namespace/local-storage created
operatorgroup.operators.coreos.com/local-operator-group created
catalogsource.operators.coreos.com/local-storage-manifests created
catalogsource.operators.coreos.com/ocs-catalogsource created
namespace/openshift-storage created
operatorgroup.operators.coreos.com/openshift-storage-operatorgroup created
subscription.operators.coreos.com/ocs-subscription created
error: no matching resources found
error: no matching resources found
error: no matching resources found
error: no matching resources found
error: no matching resources found
error: no matching resources found
error: no matching resources found
pod/ocs-operator-76d7bfc559-9qwn6 condition met
pod/rook-ceph-operator-6d5fcd8495-nz4px condition met
pod/noobaa-operator-7fcf7fb556-qn7ll condition met
pod/local-storage-operator-bcfd5765f-gkslj condition met
clusterrole.rbac.authorization.k8s.io/cluster-admin added: "ocs-operator"
clusterrole.rbac.authorization.k8s.io/cluster-admin added: "local-storage-operator"
storageclass.storage.k8s.io/mon-sc created
persistentvolume/hostpath0000 created
persistentvolume/hostpath0001 created
persistentvolume/hostpath0002 created
localvolume.local.storage.openshift.io/osd-disks created
configmap/rook-config-override created
node/openshift-master-0.dev4.kni.lab.eng.bos.redhat.com labeled
node/openshift-master-1.dev4.kni.lab.eng.bos.redhat.com labeled
node/openshift-master-2.dev4.kni.lab.eng.bos.redhat.com labeled
error: unable to recognize "STDIN": no matches for kind "StorageCluster" in version "ocs.openshift.io/v1alpha1"
```

**Comment by jtaleric** [comment]
```
[root@dhcp-19-130-242 OCS]# oc describe crd/storageclusters.ocs.openshift.io
Name:         storageclusters.ocs.openshift.io
Namespace:    
Labels:       <none>
Annotations:  <none>
API Version:  apiextensions.k8s.io/v1beta1
Kind:         CustomResourceDefinition
Metadata:
  Creation Timestamp:  2019-09-09T16:16:20Z
  Generation:          1
  Resource Version:    23008
  Self Link:           /apis/apiextensions.k8s.io/v1beta1/customresourcedefinitions/storageclusters.ocs.openshift.io
  UID:                 28a3583a-d31d-11e9-b8c5-98039b6191d0
Spec:
  Conversion:
    Strategy:  None
  Group:       ocs.openshift.io
  Names:
    Kind:       StorageCluster
    List Kind:  StorageClusterList
    Plural:     storageclusters
    Singular:   storagecluster
  Scope:        Namespaced
  Subresources:
    Status:
  Validation:
    openAPIV3Schema:
      Properties:
        API Version:
          Description:  APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources
          Type:         string
        Kind:
          Description:  Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds
          Type:         string
        Metadata:
          Type:  object
        Spec:
          Properties:
            Instance Type:
              Type:  string
            Manage Nodes:
              Type:  boolean
            Mon PVC Template:
              Type:  object
            Storage Device Sets:
              Items:
                Properties:
                  Config:
                    Type:  object
                  Count:
                    Format:  int64
                    Type:    integer
                  Data PVC Template:
                    Type:  object
                  Name:
                    Type:  string
                  Placement:
                    Properties:
                      Node Affinity:
                        Type:  object
                      Pod Affinity:
                        Type:  object
                      Pod Anti Affinity:
                        Type:  object
                      Tolerations:
                        Items:
                          Type:  object
                        Type:    array
                    Type:        object
                  Portable:
                    Type:  boolean
                  Resources:
                    Type:  object
                Required:
                  name
                  count
                  resources
                  placement
                  dataPVCTemplate
                  portable
                Type:  object
              Type:    array
          Type:        object
        Status:
          Properties:
            Conditions:
              Description:  Conditions describes the state of the StorageCluster resource.
              Items:
                Properties:
                  Last Heartbeat Time:
                    Format:  date-time
                    Type:    string
                  Last Transition Time:
                    Format:  date-time
                    Type:    string
                  Message:
                    Type:  string
                  Reason:
                    Type:  string
                  Status:
                    Type:  string
                  Type:
                    Type:  string
                Required:
                  type
                  status
                  lastHeartbeatTime
                  lastTransitionTime
                Type:  object
              Type:    array
            Related Objects:
              Description:  RelatedObjects is a list of objects created and maintained by this operator. Object references will be added to this list after they have been created AND found in the cluster.
              Items:
                Type:  object
              Type:    array
          Type:        object
  Version:             v1
  Versions:
    Name:     v1
    Served:   true
    Storage:  true
Status:
  Accepted Names:
    Kind:       StorageCluster
    List Kind:  StorageClusterList
    Plural:     storageclusters
    Singular:   storagecluster
  Conditions:
    Last Transition Time:  2019-09-09T16:16:20Z
    Message:               no conflicts found
    Reason:                NoConflicts
    Status:                True
    Type:                  NamesAccepted
    Last Transition Time:  <nil>
    Message:               the initial names have been accepted
    Reason:                InitialNamesAccepted
    Status:                True
    Type:                  Established
  Stored Versions:
    v1
Events:  <none>

```

**Comment by mcornea** [comment]
I'm also seeing the same issue when running make OCS:

```shell
[cloud-user@rhhi-node-worker-0 install-scripts]$ make OCS
set -e; pushd OCS; make; popd     
~/install-scripts/OCS ~/install-scripts
make[1]: Entering directory '/home/cloud-user/install-scripts/OCS'
./deploy-ocs.sh                   
Using osd_devices /dev/sdb,/dev/sdc,/dev/sdd,/dev/sde
namespace/local-storage created   
operatorgroup.operators.coreos.com/local-operator-group created
catalogsource.operators.coreos.com/local-storage-manifests created
catalogsource.operators.coreos.com/ocs-catalogsource created
namespace/openshift-storage created
operatorgroup.operators.coreos.com/openshift-storage-operatorgroup created
subscription.operators.coreos.com/ocs-subscription created
error: no matching resources found     
error: no matching resources found                                
error: no matching resources found
error: no matching resources found                   
error: no matching resources found                                                                             
error: no matching resources found
error: no matching resources found           
pod/ocs-operator-76d7bfc559-nzpz4 condition met
pod/rook-ceph-operator-6d5fcd8495-z8cjt condition met
pod/noobaa-operator-7fcf7fb556-8rgm7 condition met                                                                                                                                                                                           
pod/local-storage-operator-bcfd5765f-pd7hm condition met
clusterrole.rbac.authorization.k8s.io/cluster-admin added: "ocs-operator"
clusterrole.rbac.authorization.k8s.io/cluster-admin added: "local-storage-operator"
storageclass.storage.k8s.io/mon-sc created
persistentvolume/hostpath0000 created
persistentvolume/hostpath0001 created
persistentvolume/hostpath0002 created
localvolume.local.storage.openshift.io/osd-disks created
node/rhhi-node-master-0 labeled
node/rhhi-node-master-1 labeled
node/rhhi-node-master-2 labeled
error: unable to recognize "STDIN": no matches for kind "StorageCluster" in version "ocs.openshift.io/v1alpha1"
error: no matching resources found
error: no matching resources found
error: no matching resources found
error: no matching resources found
error: no matching resources found
error: no matching resources found
error: no matching resources found
error: no matching resources found
error: no matching resources found
```

**Comment by karmab** [comment]
addressed in https://github.com/openshift-kni/install-scripts/pull/109

**Comment by sreichar** [comment]
closed by #109
